### ISSUE TITLE
Question - Can i connect onpremise service fabric cluster to AAD ?

### ISSUE THREAD

**Issue by nareshkhatri81** [Azure/service-fabric-issues#1310]
Team,

is it possible to connect AAD to Onpremise cluster for authentication ?
could you please let me know steps for same if possible ?

Thanks,
Naresh Khatri

**Comment by linggengmsft** [comment]
Yes, the only difference is adding the AAD settings in the cluster config file instead of on azure portal, under security section: tenantId, clusterApplication, and clientApplication

**Comment by nareshkhatri81** [comment]
@linggengmsft 
Thanks for providing information.
one more question.
With AAD authentication, does service fabric does auditing ?
like which admin user has updated application, scaled application or delete applications.

if we have multiple admins, is there way to see what actions were performed by which admin in logs ?

Thanks,
naresh Khatri

**Comment by linggengmsft** [comment]
I don't believe there is an easy way to user, though through our system traces we would be able to tell.

**Comment by nareshkhatri81** [comment]
@linggengmsft 
its fine it user doesnt know. but from service fabric ETW logs will we be able to get information ?
our idea is that if some admin performs scaling, deleting app, it should be log somewhere.
we are planning to push ETW logs to dynatrace. so if etw logs contains those information. we are good.
can you confirm on that ?
